drun.ts

```diff
@@ -1,3 +1,4 @@
+import * as log from "https://deno.land/std@v0.53.0/log/mod.ts";
```

Comment:
Can you add this import into the `deps.ts` file and import from it like this:
`import { log } from './src/deps.ts'` (I haven't seen before that the deps.ts was not at the root, I will change that after your PR).

Reply:
Sure